cypress/package.json

```diff
@@ -34,11 +34,14 @@
     "cypress-real-events": "^1.14.0",
     "cypress-tags": "^1.2.2",
     "decompress": "^4.2.1",
+    "dotenv": "^16.0.0",
```

Comment:
This got replaced with `dotenvx` and pulling it out of the scripts that were using it.

See the `e2e:run:local` script